ci/ray_ci/tester_container.py

```diff
@@ -41,6 +45,19 @@ def __init__(
         if not skip_ray_installation:
             self.install_ray(build_type)
 
+    def _get_bazel_log_mount_dir(self) -> Tuple[str, str]:
+        """
+        Create a temporary directory in the current container to store bazel event logs
+        produced by the test runs. We do this by using the artifact mount directory from
+        the host machine as a shared directory between all containers.
+        """
+        artifact_host, artifact_container = self.get_artifact_mount()
+        bazel_log_dir_container = tempfile.mkdtemp(dir=artifact_container)
+        bazel_log_dir_host = bazel_log_dir_container.replace(
+            artifact_container, artifact_host
+        )
```

Comment:
could you do a safer prefix-based replace?

if you make up a random dir name on the previous line, you do not need this replace anymore.